index.html

```diff
@@ -36,6 +36,7 @@
             <li class="navlink"><a href="#about"><span id="open"> &lt; </span>About<span id="open"> /&gt; </span></a></li>
             <li class="navlink"><a href="/gsoc"><span id="open"> &lt; </span>GSoC<span id="open"> /&gt; </a></li>
             <li class="navlink"><a href="#projects"><span id="open"> &lt; </span>Projects<span id="open"> /&gt; </a></li>
+            <li class="navlink"><a href="https://kamandprompt.github.io/blog/"><span id="open"> &lt; </span>Blog<span id="open"> /&gt; </a></li>
```

Comment:
You don't need to put the entire link `"https://kamandprompt.github.io/blog/`,  you can just put `/blog` and it will work adding the base URL. This is because the base URL can change so keeping it relative is a good idea.